src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -253,7 +269,16 @@ export const AccountDropdown: FC<AccountDropdownProps> = ({
     handleClick,
   ])
 
+  /**
+   * do NOT remove these checks, this is not a visual thing, this is a safety check!
+   *
+   * this component is responsible for selecting the correct account for operations where
+   * we are sending funds, we need to be paranoid.
+   */
   if (!accountIds.length) return null
+  if (!accountNumber) return null
```

Comment:
This guy is going to cause issues. `accountNumber` of `0` is falsy.

Reply:
ahh, incoming